tests/test_version_providers.py

```diff
@@ -225,3 +225,98 @@ def test_scm_provider_default_without_commits_and_tags(config: BaseConfig):
     provider = get_provider(config)
     assert isinstance(provider, ScmProvider)
     assert provider.get_version() == "0.0.0"
+
+
+NPM_PACKAGE_JSON = """\
+{
+  "name": "whatever",
+  "version": "0.1.0"
+}
+"""
+
+NPM_PACKAGE_EXPECTED = """\
+{
+  "name": "whatever",
+  "version": "42.1"
+}
+"""
+
+NPM_LOCKFILE_JSON = """\
+{
+  "name": "whatever",
+  "version": "0.1.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "whatever",
+      "version": "0.1.0"
+    },
+    "someotherpackage": {
+      "version": "0.1.0"
+    }
+  }
+}
+"""
+
+NPM_LOCKFILE_EXPECTED = """\
+{
+  "name": "whatever",
+  "version": "42.1",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "whatever",
+      "version": "42.1"
+    },
+    "someotherpackage": {
+      "version": "0.1.0"
+    }
+  }
+}
+"""
+
+
+@pytest.mark.parametrize(
+    "pkg_lock_content,pkg_lock_expected,pkg_shrinkwrap_content,pkg_shrinkwrap_expected",
+    (
+        (None, None, None, None),
+        (NPM_LOCKFILE_JSON, NPM_LOCKFILE_EXPECTED, None, None),
+        (None, None, NPM_LOCKFILE_JSON, NPM_LOCKFILE_EXPECTED),
+        (
+            NPM_LOCKFILE_JSON,
+            NPM_LOCKFILE_EXPECTED,
+            NPM_LOCKFILE_JSON,
+            NPM_LOCKFILE_EXPECTED,
+        ),
+    ),
+)
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "pkg_shrinkwrap_content, pkg_shrinkwrap_expected",
    (
        (NPM_LOCKFILE_JSON, NPM_LOCKFILE_EXPECTED), 
        (None, None)
    )
)
@pytest.mark.parametrize(
    "pkg_lock_content, pkg_lock_expected",
    (
        (NPM_LOCKFILE_JSON, NPM_LOCKFILE_EXPECTED), 
        (None, None)
    )
)
```

Reply:
@Lee-W  - Applying suggestion now. Sorry, you made this suggestion earlier, and I wasn't getting it. Makes sense to me now.